indico/modules/events/persons/schemas.py

```diff
@@ -0,0 +1,21 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from marshmallow import fields
+
+from indico.core.marshmallow import mm
+from indico.modules.events.models.persons import EventPerson
+
+
+class EventPersonSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = EventPerson
+        fields = ('id', 'identifier', 'email', 'affiliation', 'full_name',
+                  'first_name', 'last_name', 'user_identifier')
+
+    full_name = fields.Function(lambda ft: f'{ft.first_name} {ft.last_name}'.strip() or 'Unknown')
+    user_identifier = fields.Function(lambda ft: f'User:{ft.user_id}' if ft.user else None)
```

Comment:
I think you can simplify this to

    user_identifier = fields.String(attribute='user.identifier')

to get the identifier from thw userlying User. Needs testing to see if it correctly handles the case where no user is present. Also, in case this triggers an extra query for each eventperson with a user (to query the user even though you only need their id) then better stick with what you have there right now.